managedtenants/bundles/docker_api.py

```diff
@@ -148,3 +142,9 @@ def _image_exists(self, image):
             # image.url_digest, calls digest which raises HTTPError
             # https://github.com/app-sre/sretoolbox/blob/master/sretoolbox/container/image.py#L119
             return False
+
+    def validate_image(self, tag):
+        image = self.client.images.get(tag)
```

Comment:
will there be a case, when images not presents?

Reply:
i dont think so and just want to confirm..

Reply:
In this case, we call this after building either a bundle image or an indexImage, so they should always be present.